site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -110,3 +115,30 @@ curl -GET "http://localhost:9358/gameserver" -H "accept: application/json"
 ```
 {"object_meta":{"creation_timestamp":"-62135596800","labels":{"agones.dev/sdk-foo":"bar"}},"spec":{"health":{}},"status":{"state":"Ready"}}
 ```
+
+## Running Local Mode in a Container
+
+Once you have your game server process in a container, you may also want to test the container build locally as well.
+
+Since the production agones-sdk binary has the `--local` mode built in, you can also use the production conatiner image
+locally as well!
+
+Since the SDK and your game server container need to share a port of `localhost`, one of the easiest ways to do that 
+is to have them both run with their networking occurring on the host, like so:
+
+In one shell run:
+
+```shell
+docker run --network=host --rm gcr.io/agones-images/agones-sdk:{{< release-version >}} --local
+```
+
+You should see a similar output to what you would if you were running the binary.
```

Comment:
Suggestion: "if you were running the binary outside of a container." (or "outside of docker.")

Reply:
> You should see a similar output to what you would if you were running the binary directly, i.e. outside a container.

WDYT?

Reply:
👍